index.js

```diff
@@ -70,5 +76,34 @@ function toggleColor(element) {
 
 /***** Deliverable 2 *****/
 
+const increaseLike = document.querySelector(".like-button")
+const likes = document.querySelector(".likes")
+
+increaseLike.addEventListener("click", function(){
+  const totalLike = parseInt(likes.textContent)
+  likes.textContent = totalLike + 1 + " Likes"
```

Comment:
Looks good! The key here is that you have to read some information from the DOM to get the current number of likes, increment it, and then write it back to the DOM. You can also do some string interpolation here (just a matter of preference): 

```js
likes.textContent = `${totalLike + 1} Likes`
```